File: app/src/debug/java/com/meiji/daily/SdkManager.java
package com.meiji.daily;

import android.content.Context;

import com.facebook.stetho.Stetho;

import okhttp3.OkHttpClient;
import okhttp3.logging.HttpLoggingInterceptor;

/**
 * Created by Meiji on 2017/5/2.
 */

public class SdkManager {
    public static void initStetho(Context context) {
        Stetho.initializeWithDefaults(context);
    }

    public static OkHttpClient.Builder initInterceptor(OkHttpClient.Builder builder) {
        HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
        interceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
        builder.addInterceptor(interceptor);
        return builder;
    }
}

File: app/src/debug/java/com/meiji/daily/SdkManager.kt
package com.meiji.daily

import android.content.Context

import com.facebook.stetho.Stetho
import com.meiji.daily.util.HttpLoggingInterceptor

import okhttp3.OkHttpClient

/**
 * Created by Meiji on 2017/5/2.
 */

object SdkManager {
    fun initStetho(context: Context) {
        Stetho.initializeWithDefaults(context)
    }

    fun initInterceptor(builder: OkHttpClient.Builder): OkHttpClient.Builder {
        val interceptor = HttpLoggingInterceptor()
        interceptor.mLevel = HttpLoggingInterceptor.Level.BODY
        builder.addInterceptor(interceptor)
        return builder
    }
}
